Setup.bat

```diff
@@ -111,6 +111,7 @@ call :MarkEndOfBlock "Copy GDK schema"
 
 call :MarkStartOfBlock "Build C# utilities"
     %MSBUILD_EXE% /nologo /verbosity:minimal .\SpatialGDK\Build\Programs\Improbable.Unreal.Scripts\Improbable.Unreal.Scripts.sln /property:Configuration=Release
+    %MSBUILD_EXE% /nologo /verbosity:minimal .\SpatialGDK\ExternalSchemaCodegen\Programs\Codegen.sln /restore /property:Configuration=Release

```

Comment:
Why don't we keep this solution in the build/programs folder? Maybe we could add the codegen project to the same solution and rename the solution to Improbable.Unreal.Tools or similar?

Reply:
I assumed the `Build` directory was explicitly related to the `BuildWorker` executable and related libs, hence I chose to put the bat file in the same subdirectory as the code generator.

Reply:
Happy to move into the Build directory / .sln if you think this is preferrable?

Reply:
I think it should be in the same solution but might be worth getting the GDK teams opinion too

Reply:
Yes, please add it to the Unreal.Scripts solution